index.ts

```diff
@@ -417,34 +379,39 @@ class Analytics {
       });
   }
 
-  _exponentialDelay(attempt, _error, _response) {
-    const delay = Math.pow(2, attempt + 1) * 200;
-    const randomSum = delay * 0.2 * Math.random(); // 0-20% of the delay
-    return delay + randomSum;
+  /**
+   * Calculates the amount of time to wait before retrying a request, given the number of prior
+   * retries (excluding the initial attempt).
+   *
+   * @param priorRetryCount the number of prior retries, starting from zero
+   */
+  private getExponentialDelay(priorRetryCount: number): number {
+    const delay = 2 ** priorRetryCount * 200;
+    const jitter = delay * 0.2 * Math.random(); // 0-20% of the delay
```

Comment:
spicy rename

Reply:
This value isn't a sum, it's an input to a sum